tests/test-cases/Group23-VIC-Machine-Service/23-01-Metadata.robot

```diff
@@ -22,13 +22,14 @@ Default Tags
 
 
 *** Keywords ***
-Get Version
-    ${out}=  Run  netstat -l | grep 1337
-    Log  ${out}
-    ${out}=  Run  ps aux | grep vic-machine-server
+Diagnose Get Version
+    ${out}=  Run  ps aux | grep ${VIC_MACHINE_SERVER_BIN}
```

Comment:
Is this a basic assertion that the server is still running? You might rename it to `Assert Server Running` or add a comment. It took me a surprisingly long time to figure out what the intent was.

Reply:
This keyword doesn't actually assert anything; it just provides additional diagnostic information in case the `Get Version` keyword fails. I'm not actually convinced this is very useful...

Reply:
Maybe it _should_ assert that the process is running? Maybe this should be part of `Test Setup` so we're asserting that the expected service is present.

Reply:
With tests running in parallel, there aren't a lot of meaningful things we can assert here.

I'm just going to remove this.